source/includes/usage-examples/connect-sample-app.cpp

```diff
@@ -0,0 +1,29 @@
+#include <mongocxx/instance.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/exception/exception.hpp>
+#include <iostream>
+#include <bsoncxx/json.hpp>
+
+int main()
+{
+    mongocxx::instance instance{};
```

Comment:
Our code style guide that was developed with @kevinAlbs for this standardization instructs us to create the instance with `{}`

Reply:
Recommend updating the code style guide to avoid redundant `{}`, which may confuse users and incorrectly imply its absence may produce an uninitialized variable.

Reply:
Added suggestion to [C/C++ Driver Code Example Patterns and Style Reference](https://docs.google.com/document/d/1wBsUnT69a7XNTUOZd1agQae_pL1fREl-mcg34QgYaCs/edit#heading=h.afeu4uin7cta)

Reply:
Updated for this example. I'll open a separate PR to remove other the curly brackets in our other examples throughout the repo